docs/java/java-gui.md

```diff
@@ -0,0 +1,27 @@
+# Working with GUI application in Visual Studio Code
+
+You can develop Java GUI applications in Visual Studio Code easily. To achieve that, you need to install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes all the required extensions to develop Java GUI application.
+
+If you run into any issues when using the features below, you can contact us by entering an [issue](https://github.com/microsoft/vscode-java-pack/issues).
+
+## Develop JavaFX Application
+
+Now you can simply create a new JavaFX application will very few steps in Visual Studio Code. Open the command palette **Command Palette**  (`kb(workbench.action.showCommands)`) and then select the command: **Java: Create Java Project**. You will find the option **JavaFX** in the list, which will help you scaffold a new JavaFX project via Maven archetype.
```

Comment:
```suggestion
Now you can simply create a new JavaFX application with just a few steps in Visual Studio Code. Open the command palette **Command Palette**  (`kb(workbench.action.showCommands)`) and then select the command: **Java: Create Java Project**. You will find the option **JavaFX** in the list, which will help you scaffold a new JavaFX project via Maven archetype.
```

Reply:
also suggest to break this content down into several steps:
Step 1: Install extension pack 
Step 2: xx
Step 3: yy